addon/components/bulma-radio.js

```diff
@@ -12,16 +12,21 @@ export default BulmaInput.extend({
 
   classNames: ['radio'],
   type: 'radio',
+
   // Bindings are not comprehensive. More complex implementations should use a native element with classes applied
   classNameBindings: [
-    'capture',
-    'checked',
-    'list'
+    'checked'
   ],
 
   init() {
     this._super(...arguments);
 
+    let defaultAttrBindings = get(this, 'attributeBindings').slice(0);
```

Comment:
@jbailey4 Nit pick...`slice` begin param defaults to zero. You can just do `slice()`